### PR TITLE
Task 7 - POST complete

### PR DESCRIPTION
**Pull request by souste**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -186,3 +186,58 @@ describe("/api/reviews/:review_id/comments", () => {
...
+        userName: "philippaclaire9",
+        body: "Hello there!",
+      })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toEqual({
+          author: "philippaclaire9",
+          body: "Hello there!",
+          votes: expect.any(Number),
+          review_id: expect.any(Number),
```
Comment (yalicee) [inline __tests__/app.test.js:205]: we know the review id is 3

**Review by yalicee — CHANGES_REQUESTED** [review]
Nice work but you need to add more error handling for when the username is not found and when there is a missing required field in the body

File: __tests__/app.test.js (hunk 1 above)
Comment (souste) [reply __tests__/app.test.js:205]: Thanks Alex I will make the changes.

File: model/model.js [hunk 2]
```diff
@@ -46,6 +46,19 @@ exports.selectReviewIdComments = (review_id) => {
...
 };
+
+exports.insertComment = (review_id, comment) => {
+  const { userName, body } = comment;
+  const queryStr = `INSERT INTO comments (review_id, author, body) VALUES ($1, $2, $3) RETURNING *;`;
+  // [review_id];
+  const queryValues = [review_id, userName, body];
+
+  return db.query(queryStr, queryValues).then((result) => {
+    console.log(result.rows[0]);
```
Comment (hevdevs) [inline model/model.js]: remove console.log before merge

**Review by hevdevs — APPROVED** [review]
Nice work Stephen! Looks good to me, just give it a scan and make sure to remove any console.logs and any unnecessary comments before merging

File: model/model.js (hunk 2 above)
Comment (souste) [reply model/model.js]: Thanks Hev, I'll make the changes!

**Pull request merged**